apps/dhparam.c

```diff
@@ -148,11 +148,17 @@ int dhparam_main(int argc, char **argv)
             break;
         }
     }
+
+    /* One optional argument, bitsize to generate. */
```

Comment:
````
openssl dhparam 
````
Gives an error for me locally, since it requires the generator (e.g -2) to be set in order to set a default bitsize.. That is a bit weird.

Reply:
I get an error from 1.1.1 "can't load params" 3.0 plain and this branch do the same thing.  What error do you see that is unexpected?

Reply:
The point is it says that the bitsize is an optional parameter. So if you dont specify it, it should just choose a default one. This is not what it does - it exits unless you also specify another optional param i.e '-2'.
So it should either
(1) choose a default generator + bitsize OR
(2) print an error specifying that one of the generator params is required..

Reply:
I didn't change the behavior about this stuff. It looks to me (line 163/164) that if you're generating something it does use a default bitsize. The manpage describes behavior that is weird.  But I don't think changing the way it works is something to do in this PR.